docs/deployment.md

```diff
@@ -113,12 +113,16 @@ Zeit [Now](http://zeit.co/now) is a great service for running Probot apps. After
 
 1. Clone the app that you want to deploy. e.g. `git clone https://github.com/probot/stale`
 
-1. Run `now` to deploy, replacing the `APP_ID` and `WEBHOOK_SECRET` with the values for those variables, and setting the path for the `PRIVATE_KEY`:
+1. Add a `now-start` script to `package.json` as follows:
+
+        "now-start": "PRIVATE_KEY=$(echo $PRIVATE_KEY_BASE64 | base64 -d) npm start"
+
+1. Run `now` to deploy, replacing the `APP_ID` and `WEBHOOK_SECRET` with the values for those variables, and setting the `PRIVATE_KEY_BASE64`:
 
         $ now -e APP_ID=aaa \
             -e WEBHOOK_SECRET=bbb \
             -e NODE_ENV=production \
-            -e PRIVATE_KEY="$(cat ~/Downloads/*.private-key.pem)"
+            -e PRIVATE_KEY_BASE64="$(cat ~/Downloads/*.private-key.pem | base64)"
```

Comment:
Hey don’t think that probot supports `PRIVATE_KEY_BASE64` yet? Or am I missing something? But if it doesn’t, I like the idea :)

As a workaround, you can do this: https://github.com/wip/app/blob/76070f4aecd7d375c9c86ba7885ac24fd16f8fa9/package.json#L13

```json
  "scripts": {
    "start": "probot run ./index.js",
    "now-start": "PRIVATE_KEY=$(echo $PRIVATE_KEY | base64 -d) npm start"
  }
```

Reply:
Please look a few lines above. : D

Reply:
🤦‍♂️  okidoke